backend/scripts/mirror_s3_data.sh

```diff
@@ -41,20 +41,30 @@ if [[ -n "$WMG_CUBE" ]]; then
 fi
 
 
-echo Mirroring S3 Dataset data from $SRC_ENV to $DEST_ENV...
+if [[ $DEST_ENV == 'rdev' && -n "$CELLGUIDE" ]]; then
+  # Copy CellGuide data
+  echo "Copying CellGuide snapshot..."
 
-if [[ $DEST_ENV == 'rdev' ]]; then
+  latest_snapshot=$(aws s3 cp s3://cellguide-data-public-${SRC_ENV}/latest_snapshot_identifier -)
+  echo -n $latest_snapshot | aws s3 cp - s3://cellguide-data-public-dev/env-rdev-cellguide/${STACK}/latest_snapshot_identifier
+  $S3_SYNC_CMD s3://cellguide-data-public-${SRC_ENV}/${latest_snapshot} s3://cellguide-data-public-dev/env-rdev-cellguide/${STACK}/${latest_snapshot}
```

Comment:
Add a comment re: the cellguide bucket hack, i..e, it's not the expected bucket name for a rdev env and why (cloudfront)

Reply:
✔️